teach_base/lesson_02/04_vowels_hash.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Fill the hash with vowels, where the value will be ordinal
+# letter number in the alphabet (a - 1).
+
+vowels_arr = %w[a e i o u y]
+vowels_hash = {}
+
+('a'..'z').each_with_index do |key, value|
```

Comment:
Your solution works, but here you pass arguments in a wrong order (you mistake key and value).
And you need to do extra `invert` operation in your case.
It will confuse developers who will try to add changes to your code.
Check the documentation about each_with _index method and do it right:
https://apidock.com/ruby/v2_5_5/Enumerable/each_with_index